torch/_dynamo/output_graph.py

```diff
@@ -313,6 +320,12 @@ def restore_graphstate(self, state: OutputGraphState):
                 removed_nodes += 1
         log.debug(f"restore_graphstate: removed {removed_nodes} nodes")
 
+    def add_grapharg(self, arg: GraphArg):
+        curr_pos = len(self.graphargs)
```

Comment:
worth renaming self.graphargs -> self._graphargs just to prove it isn't still directly used anywhere?

Reply:
We iterate over it and such, I can do this in a followup, I try to not conflate small fixes/renames with new code